### PR TITLE
Fix doc example for asciidoc

### PR DESCRIPTION
**Pull request by guillep2k**
Fix what looks like a typo to me.

### CODE REVIEW

**Comment by jolheiser** [comment]
According to [sample.ini](https://github.com/go-gitea/gitea/blob/master/custom/conf/app.ini.sample#L873) and the [code](https://github.com/go-gitea/gitea/blob/master/modules/setting/markup.go#L59) `false` is the correct value here.

**Comment by jolheiser** [comment]
Ah, shoot. You are correct, I was thinking this was explaining defaults.  
In that case, shouldn't [this line](https://github.com/go-gitea/gitea/pull/9072/files#diff-07e54da65506d1528f7f446c22a6f723R567) also be set to `true`?

**Review by lunny — APPROVED** [review]

**Comment by guillep2k** [comment]
@jolheiser In that line I think the default should be shown. Otherwise the line below should come pre-filled? 😄 But I improved the comment a little to prevent confusions.

**Review by jolheiser — APPROVED** [review]
That could be, perhaps I just interpreted it incorrectly. Either way, LGTM.

File: docs/content/doc/advanced/config-cheat-sheet.en-us.md [hunk 1]
```diff
@@ -558,13 +558,13 @@ Gitea can support Markup using external tools. The example below will add a mark
...
 [markup.asciidoc]
-ENABLED = false
+ENABLED = true
 FILE_EXTENSIONS = .adoc,.asciidoc
 RENDER_COMMAND = "asciidoc --out-file=- -"
 IS_INPUT_FILE = false
 ```
 
-- ENABLED: **false** Enable markup support.
+- ENABLED: **false** Enable markup support; set to **true** to enable this function.
```
Comment (lafriks) [inline docs/content/doc/advanced/config-cheat-sheet.en-us.md]:
```suggestion
- ENABLED: **false** Enable markup support; set to **true** to enable this renderer.
```

**Review by lafriks — CHANGES_REQUESTED** [review]
as per comment

**Comment by guillep2k** [comment]
@lafriks done

**Review by lafriks — APPROVED** [review]

**Pull request merged**